connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -864,6 +864,7 @@ protected void tick() {
             Vector3d position = collisionManager.adjustBedrockPosition(playerEntity.getPosition(), playerEntity.isOnGround());
             // A null return value cancels the packet
             if (position != null) {
+                confirmTeleport(position);
```

Comment:
Why this change?

Reply:
Without this line, you start racking up unconfirmed teleports if you execute the command `/tp ~ ~ ~` without moving anything.

Reply:
This sounds like an incorrect solution to a problem - to my knowledge, Java only confirms teleports after the server requests it, not during its tick cycle. Our system is a little different since it's designed to ensure that the Bedrock player is at the desired location first. Perhaps Bedrock doesn't send back teleports if the Bedrock server's teleport position is close to its existing location? If so, we should move that logic to JavaPlayerPositionRotationTranslator instead.

Reply:
It looks like you are correct. Teleports under 0.01 in distance are not sent back from Bedrock. I've moved the logic in the latest commit.

Reply:
Can you test that logic with rotation teleports? Not sure if there's an easy way to do that with vanilla.

Reply:
Are you referring to https://wiki.vg/Protocol#Face_Player? This packet doesn't seem to be translated at all by Geyser.

Reply:
No, I mean if the PlayerPositionRotationPacket sends a rotation value. How does Bedrock treat that?

Reply:
If you send a packet with only yaw changes (`/tp ~ ~ ~ 0 0` then`/tp ~ ~ ~ 100 0`), there is no effect on bedrock and it doesn't get sent back. Pitch changes less than 3-4 degrees, are also not sent back.

Reply:
Those sound like conditions we'd also need to add in the new if statement, if you want to add that to this PR?

Reply:
Added pitch checks in the latest commit.